### PR TITLE
Add UI tests to abtesting [SwiftUI]

### PR DESCRIPTION
**Pull request by jrcrespoh**
Add UI tests for SwiftUI version of A/B Testing Quickstart. Currently only testing static UI independent of A/B Testing & Remote Config functionality.

### CODE REVIEW

**Review by morganchen12 — COMMENTED** [review]
LGTM, but please wait for Paul's approval and discuss with Paul/iCore on what testing needs are appropriate.

File: .github/workflows/abtesting.yml
Comment (paulb777) [inline .github/workflows/abtesting.yml:57]: One nit: Change line 48 to Build and Test SwiftUI

**Review by paulb777 — APPROVED** [review]

**Pull request merged**
